js/web/script/test-runner-cli.ts

```diff
@@ -6,579 +6,583 @@
 
 import {spawnSync} from 'child_process';
 import * as fs from 'fs-extra';
-import * as globby from 'globby';
 import {default as minimatch} from 'minimatch';
 import npmlog from 'npmlog';
 import * as os from 'os';
 import * as path from 'path';
-import stripJsonComments from 'strip-json-comments';
 import {inspect} from 'util';
 
 import {bufferToBase64} from '../test/test-shared';
 import {Test} from '../test/test-types';
 
 import {parseTestRunnerCliArgs, TestRunnerCliArgs} from './test-runner-cli-args';
 
-npmlog.info('TestRunnerCli', 'Initializing...');
+async function main() {
+  const {globbySync} = await import('globby');
+  const stripJsonComments = (await import('strip-json-comments')).default;
```

Comment:
the github diff-tool is definitely doing something wrong. the only actual changes from line 24 to line 584 are:
- remove `process.exit();` at line 160
- code format done by eslint at line 325 and line 494 because of it reaches max line length

Reply:
same to e2e/run.js for adding `const {globbySync} = await import('globby');`